identity/identity/identity.go

```diff
@@ -189,7 +189,8 @@ func (id *ID) ServerTLSConfig() *tls.Config {
 	tlsConfig := &tls.Config{
 		GetCertificate: id.GetServerCertificate,
 		RootCAs:        id.ca,
-		ClientAuth:     tls.RequireAnyClientCert,
+		ClientCAs:      id.ca,
```

Comment:
This will affect the controller's ability to manually handle client certificates.

It will break:
- 3rd party cas
- currently expires client certs (recent ramco issue that we are working towards a long term solution for)

This change cannot go through.

Reply:
We should probably talk about how we want to handle this. The way the setup works now is that this code expects the implementor to override these defaults.